app/views/tutorials/show.html.erb

```diff
@@ -11,18 +11,19 @@
     <% end %>
   </ul>
 </div>
-
-<div class="col col-8">
-  <div class="title-bookmark">
-    <h3><%= @facade.current_video.title %></h3>
-    <div class="bookmarks-btn">
-      <% if current_user %>
-      <%= button_to "Bookmark", user_videos_path(video_id: @facade.current_video.id, user_id: current_user.id), method: :post, class: "btn btn-outline mb1 teal" %>
-      <% else %>
-      <%= link_to "Bookmark", login_path, class: "btn btn-outline mb1 teal" %>
-      <% end %>
+<% if !@facade.current_video.nil? %>
```

Comment:
Can you remind me what was going on here? Was it erroring if there were no videos?